src/SFA.DAS.LevyTransferMatching.Api/Controllers/PledgeController.cs

```diff
@@ -20,13 +22,35 @@ public PledgeController(IMediator mediator)
 
         [HttpGet]
         [Route("pledges")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
         public async Task<IActionResult> GetPledges()
         {
             var result = await _mediator.Send(new GetPledgesQuery());
 
             return new OkObjectResult(result.Select(x => (PledgeDto)x));
         }
-		
+
+        [HttpGet]
+        [Route("pledges/{pledgeId}")]
+        [ProducesResponseType((int)HttpStatusCode.OK)]
+        [ProducesResponseType((int)HttpStatusCode.NotFound)]
+        public async Task<IActionResult> GetPledge(int pledgeId)
```

Comment:
As discussed on web PR, this should be moved to an Opportunities controller so that the outer api mirrors the web, and we can move to a one-api-call-per-page pattern.

Reply:
This has been actioned.